Toggl.Core.Tests/UI/Transformations/TimeEntriesFlatenningTests.cs

```diff
@@ -326,6 +326,7 @@ private static IEnumerable<LogItemViewModel> expanded(IThreadSafeTimeEntry[] gro
                 representedTimeEntriesIds: group.Select(timeEntry => timeEntry.Id).ToArray(),
                 visualizationIntent: visualizationIntent,
                 isBillable: sample.Billable,
+                hasArchivedProject: sample.Project?.Active == false,
```

Comment:
You can use the null coalescing operator here, I think it reads better.

Reply:
To help with this:
```suggestion
                hasArchivedProject: !sample.Project?.Active ?? false,
```

`??` has this kind of conventional reading as a `default in case of null`, especially in combination with a null-propagation operator. I believe this is the only reason why C# devs would find this more readable than what you provided initially. Otherwise @semanticer 's solution is completely correct.

Reply:
I was trying elvis operator `?:` from Kotlin but it wasn't there so I thought there is no such feature in C#